sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetFileFormat.scala

```diff
@@ -202,8 +214,18 @@ class GpuParquetFileFormat extends ColumnarFileFormat with Logging {
 
     val conf = ContextUtil.getConfiguration(job)
 
+    val outputTimestampType = sparkSession.sessionState.conf.parquetOutputTimestampType
     val dateTimeRebaseException = "EXCEPTION".equals(
-        sparkSession.sqlContext.getConf(ShimLoader.getSparkShims.parquetRebaseWriteKey))
+      sparkSession.sqlContext.getConf(ShimLoader.getSparkShims.parquetRebaseWriteKey))
+    // prior to spark 311 int96 don't check for rebase exception
+    // https://github.com/apache/spark/blob/068465d016447ef0dbf7974b1a3f992040f4d64d/sql/core/src/
+    // main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetWriteSupport.scala#L195
+    val isBeforeSpark311 = ShimLoader.getSparkShims.isBeforeSpark311
```

Comment:
This misses the point of my previous comment.  Some Spark platforms could be a _hybrid of Spark changes_ such that a single version check is not appropriate.  As in this case -- we really don't care what version of Spark it is, we're trying to understand whether INT96 has its own set of configs or not.  That's the predicate method we need, a way to tell if INT96 has been separated from datetime.  That's going to be more reliable than checking the Spark version number on Spark platforms that have cherry-picked changes from future Spark versions beyond what version they are actually reporting.